OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -55,6 +55,8 @@ void Millard2012EquilibriumMuscle::constructProperties()
     constructProperty_ForceVelocityCurve(ForceVelocityCurve());
     constructProperty_FiberForceLengthCurve(FiberForceLengthCurve());
     constructProperty_TendonForceLengthCurve(TendonForceLengthCurve());
+
+    setMinControl(get_minimum_activation());
```

Comment:
:warning: The `minimum_activation` property can be modified in `extendFinalizeFromProperties()`. I think `setMinControl();` needs to be there (e.g., at L123). Perhaps something like
```cpp
setMinControl( std::max(get_minimum_activation(), get_min_control()) );
```
in case the user provided a non-default value for `min_control`?